installer/asgi.py

```diff
@@ -0,0 +1,22 @@
+# -*- encoding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import fastapi
+from fastapi.staticfiles import StaticFiles
+
+
+application = fastapi.FastAPI()
+application.mount(
+    "/", StaticFiles(directory="/app/installer/build", html=True), name="installer"
+)
```

Comment:
Should this be placed in a installer directory so it's clear it's not the engine itself?